samples/dotnet/KernelHttpServer/KernelHttpServer.csproj

```diff
@@ -14,23 +7,21 @@
     <Nullable>enable</Nullable>
     <ImplicitUsings>disable</ImplicitUsings>
     <IsPackable>false</IsPackable>
+    <LangVersion>10</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.13" />
```

Comment:
Wouldn't these version numbers belong more in the Directory.Packages.props file?

Reply:
Normally, yes. In the /samples/ though, we want to allow developers to copy a sample folder out of the repo and run it on its own, with all dependencies self-contained or available via nuget. In the Directories.* files under samples, we're only really adding analyzers, which are optional and help standardize formatting across the repo (but someone copying the project out and experimenting with it on its own, probably doesn't need all of our analyzers). 

LangVersion, on the other hand, if not specified, will actually break the build, which is why it's specified directly in each project. On that note, I will remove these props from the common props so that they're not accidentally omitted from future samples.